data/scenarios/Challenges/drill_test.yaml

```diff
@@ -1,113 +0,0 @@
-name: Test drill
```

Comment:
@byorgey could you move it to tests instead, please? If I have to, I will write down a solution.  :smile:

Reply:
Good idea, will do!